webrtc-c/canary/src/CloudwatchMonitoring.cpp

```diff
@@ -29,8 +32,15 @@ VOID CloudwatchMonitoring::deinit()
 VOID CloudwatchMonitoring::push(const MetricDatum& datum)
 {
     Aws::CloudWatch::Model::PutMetricDataRequest cwRequest;
+    MetricDatum single = datum;
+    MetricDatum aggregated = datum;
+
+    single.AddDimensions(this->channelDimension);
+    aggregated.AddDimensions(this->labelDimension);
```

Comment:
I like this idea but perhaps it would be better to add the label dimension to the single datum as well.  This way the label information available is while looking based on the single datum and if needed you can correlate that to the aggregated datum items.

Reply:
Yep, that makes sense.